docs/naps/9-multiple-canvases.md

```diff
@@ -164,8 +164,10 @@ Slice state for each layer is currently stored on the Layer model. Again, see [N
 > Also - the various `_<Layer>SliceResponse` classes introduced by async slicing may already fill much of this role. Another option to consider here is to codify a protocol ([`typing.Protocol`](https://docs.python.org/3/library/typing.html#typing.Protocol)) for these classes. Even this protocol may not be necessary - early prototypes use these classes as-is with minimal modifications.
 > [name=Ashley A]
 
+(nap_9_design_and_architecture)
+
```

Comment:
```suggestion
(nap_9_design_and_architecture)=
```
Right now the link in the footer doesn't work, because this renders like this:
<img width="354" alt="image" src="https://github.com/napari/docs/assets/76622105/067061ed-f2a4-4348-9211-8bb39dbc12ea">


 I think this should fix it.

Reply:
:facepalm: I must have done this half asleep, will fix!

Reply:
no worries 😄 
and no rush!

Reply:
Actually we can just use the old label, what doesn't work is the capitalization. So we can just change the reference to read `(#ui-design-and-architecture)`